### ISSUE TITLE
filter out needs-traige?

### ISSUE THREAD

**Issue by goern** [thoth-station/thoth-application#2363]
Shall we filter out all the `needs-triage` issues, as that are in a different lifecycle anyways?

https://github.com/thoth-station/thoth-application/blob/cf6eeb185d1642958271b129a06929279c6d56dc/triage-party/overlays/moc/config.yaml#L190

/kind feature
/priority critical-urgent
/assign @codificat

**Comment by codificat** [comment]
I am more inclined to essentially ignore the `needs-triage` label altogether in triage-party rules, by relying only on the presence / lack of a `triage/*` label. I believe this would help us fix #2359.

That particular rule already looks for missing `triage/*` labels: https://github.com/thoth-station/thoth-application/blob/cf6eeb185d1642958271b129a06929279c6d56dc/triage-party/overlays/moc/config.yaml#L190-L196

so, filtering also with `needs-triage` would be, IMO, redundant or inconsistent - there is a rule to check for the inconsistency: https://github.com/thoth-station/thoth-application/blob/cf6eeb185d1642958271b129a06929279c6d56dc/triage-party/overlays/moc/config.yaml#L637-L643

In other words: my understanding is that any issue that does not have any `triage/*` label:
- is in the same situation as one that has `needs-triage`
- needs to be triaged (and assigned a `triage/*` label) regardless of the presence of a `needs-triage` label

Having said that, what I would suggest for this rule that looks for untriaged issues is to move it out of the daily triage collection. I believe it can be folded into this weekly view rule: https://github.com/thoth-station/thoth-application/blob/cf6eeb185d1642958271b129a06929279c6d56dc/triage-party/overlays/moc/config.yaml#L308

/triage accepted
/lifecycle active
